src/System.Management.Automation/singleshell/config/MshSnapinInfo.cs

```diff
@@ -1039,11 +1039,7 @@ internal static Collection<PSSnapInInfo> ReadEnginePSSnapIns()
 
                 string moduleName = Path.Combine(applicationBase, defaultMshSnapinInfo.AssemblyName + ".dll");
 
-                if (File.Exists(moduleName))
-                {
-                    moduleName = Path.Combine(applicationBase, defaultMshSnapinInfo.AssemblyName + ".dll");
-                }
-                else
+                if (!File.Exists(moduleName))
```

Comment:
Is this change related?

Reply:
Also, do we have any wildcard tests?

Reply:
This change is just a quick cleanup of something that I ran into :) It does `moduleName = Path.Combine(applicationBase, defaultMshSnapinInfo.AssemblyName + ".dll");` twice unnecessarily.

Reply:
As for tests, I don't think we have any tests explicitly for `WildcardPattern`, but it's been indirectly exercised by all other tests.